src/dotnet/Users.Contracts/UserConstants.cs

```diff
@@ -14,4 +14,11 @@ public static class Claims
     {
         public static string Status => "urn:actual.chat:status";
     }
+
+    public static class Presence
+    {
+        // TODO: move this to configurable options when client side settings API is implemented
```

Comment:
There is local settings API - Kvas

Reply:
Hi, sorry, this is done, and done a bit better in terms of constants & actual timings... I didn't look into this PR when I started, sorry. I migrated one thing that was missing in my case (extra option).